recipes/xdoctest/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "xdoctest" %}
+{% set version = "0.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Erotemic/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 477b4e8fb6c3940a2b293d8324cb690bff70d77a6e383ff3ccdbddc6b958ddcf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
This recipe should be `noarch`, so please add `noarch: python` back.

Reply:
I think `noarch` doesn't work here because of the entry point that the `setup.py` defines. On Windows a binary named xdoctest.exe is created, see https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=124314&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=78a0099b-3f13-5ae9-c1f0-24f71d1eac20&l=787

Reply:
Nevermind, `noarch` seems to work after all if I explicitly define the entry point in the recipe.